ado-pipeline/build/docker-build.yaml

```
@@ -0,0 +1,30 @@
+parameters:
+  - name: dockerFilePath
+    type: string
+    default: "Dockerfile"
+steps:
+  - task: Docker@2
+    displayName: Login to ACR
+    inputs:
+      command: login
+      containerRegistry: AZD-FCP-ACR-SND2
+
+  - bash: |
+      if [ $isMainBranch == "true" ]; then
```

```suggestion
      if [ "$(isMainBranch)" == "true" ]; then
```
You have set it like this [here](https://github.com/DEFRA/fcp-pipeline-common/blob/PL-658-docker-image/ado-pipeline/init/app-version.yaml#L34) should it be the same style?